cmd/application.go

```diff
@@ -53,17 +51,18 @@ var applicationCreateCmd = &cobra.Command{
 	Args: cobra.ExactArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
 		// Args validation makes sure that there is exactly one argument
-		name := args[0]
-		// validate application name
-		err := validateName(name)
+		appName := args[0]
+		// validate application appName
+		err := validateName(appName)
 		checkError(err, "")
 		client := getOcClient()
-		fmt.Printf("Creating application: %v\n", name)
-		err = application.Create(client, name)
+		getAndSetNamespace(client)
+		fmt.Printf("Creating application: %v\n", appName)
+		err = application.Create(client, appName)
 		checkError(err, "")
-		err = application.SetCurrent(client, name)
+		err = application.SetCurrent(client, appName)
 		checkError(err, "")
-		fmt.Printf("Switched to application: %v\n", name)
+		fmt.Printf("Switched to application: %v\n", appName)
```

Comment:
Can we say in which project, if current project is not same as the project in which app was created?

Reply:
updated :)